spec/views/api_new/transactions/index.json.jbuilder_spec.rb

```diff
@@ -22,7 +22,7 @@ def base_url(nonprofit_id, transaction_id)
 	end
 
 	subject(:json) do
-		view.lookup_context.prefixes = view.lookup_context.prefixes.drop(1)
+		view.lookup_context.prefixes = view.lookup_context.prefixes.drop(2) # Rails does weird things in view specs when you use a route namespace
```

Comment:
The Rails view specs have been consistently weird when I've tried to test the views inside a route namespace. Why is drop(2) needed now? Beats me but something changed in Rails 6.1.